components/automate-ui/src/app/pages/api-token/list/api-token-list.component.html

```diff
@@ -63,7 +63,7 @@
                 <span *ngIf="token.projects.length === 1">{{ token.projects[0] }}</span>
                 <span *ngIf="token.projects.length > 1">{{ token.projects.length }} projects</span>
               </chef-td>
-              <chef-td>{{ token.created_at | datetime:'ddd, MMMM D, YYYY [at] H:mm a' }}</chef-td>
+              <chef-td>{{ token.created_at | datetime:'ddd, DD MMM YYYY HH:mm:ss [UTC]' }}</chef-td>
```

Comment:
Same formatting again - just happening in html file instead of component here.  Felt appropriate to leave the operation in the place it already existed.